### PR TITLE
Give Logo a link option

### PR DESCRIPTION
**Pull request by julieg18**

### CODE REVIEW

File: src/components/Logo/Logo.css [hunk 1]
```diff
@@ -5,6 +5,17 @@
...
+  border: 0;
+  clip: rect(0 0 0 0);
+  height: 1px;
+  margin: -1px;
+  overflow: hidden;
+  padding: 0;
+  position: absolute;
+  width: 1px;
+}
+
```
Comment (iceicery) [inline src/components/Logo/Logo.css:18]: Nice add! It is very nice that you always think about this.

**Review by iceicery — APPROVED** [review]
Nice job! The change is perfect. Do you think we need to add some hover style if it is a link?

**Comment by julieg18** [comment]
Good question. I considered it but I couldn't think of a style that worked... Do you have any ideas for the hover style?

**Comment by iceicery** [comment]
I guess a change of opacity and a pointer would work. Make it look similar
to the other link at nav bar. What do you think?

Julie <notifications@github.com> 於 2021年1月20日 週三 下午1:39寫道：

**Comment by julieg18** [comment]
I'll send you how it looks on slack.

**Pull request merged**